backend/lib/endpoints/organizations.js

```diff
@@ -0,0 +1,95 @@
+const httpErrors = require("http-errors");
+
+const {
+  createOrganizationSchema,
+  deleteOrganizationSchema,
+  getOrganizationSchema,
+  getOrganizationsSchema,
+  updateOrganizationSchema,
+} = require("./schema/organizations");
+
+/*
+ * /api/organizations
+ */
+async function routes(app) {
+  const Organization = app.mongo.model("Organization");
+
+  app.delete(
+    "/:organizationId",
+    {
+      preValidation: [app.authenticate],
+      schema: deleteOrganizationSchema,
+    },
+    async (req) => {
+      // TODO: make sure user can only delete organizations they own
+      return Organization.findByIdAndRemove(req.params.organizationId);
+    },
+  );
+
+  app.get(
+    "/",
+    {
+      preValidation: [app.authenticate],
+      schema: getOrganizationsSchema,
+    },
+    async (req, reply) => {
+      const { ownerId } = req.params;
+      const filter = ownerId ? { ownerId } : {};
+      const sortedOrganizations = await Organization.find(filter).sort({
+        name: 1,
+      });
+      return reply.send(sortedOrganizations);
+    },
+  );
+
+  app.get(
+    "/:organizationId",
+    {
+      preValidation: [app.authenticate],
+      schema: getOrganizationSchema,
+    },
+    async (req) => {
+      const result = await Organization.findById(req.params.organizationId);
+      if (result === null) {
```

Comment:
not major but we could make this a one liner, maybe something like 

`!result ? new httpErrors.NotFound() : ""`

Reply:
I don't have a strong preference, I was just keeping it consistent with the route handlers in the other endpoints.